src/app/core/services/netsuite/netsuite-configuration/netsuite-import-settings.service.ts

```
@@ -1,23 +1,93 @@
+
+  static constructCustomSegmentPayload(customSegmentForm: FormGroup, workspaceId: number): CustomSegment {
+    return {
+      segment_type: customSegmentForm.get('customFieldType')?.value,
+      script_id: customSegmentForm.get('scriptId')?.value,
+      internal_id: customSegmentForm.get('internalId')?.value,
+      workspace: workspaceId
+    };
+  }
+
+  static mapAPIResponseToFormGroup(importSettings: NetsuiteImportSettingGet | null, netsuiteFields: IntegrationField[], destinationAttribute: DestinationAttribute[]): FormGroup {
+    const expenseFieldsArray = importSettings?.mapping_settings ? this.constructFormArray(importSettings.mapping_settings, netsuiteFields) : [];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)